src/Controller/StoreApiController.php

```diff
@@ -51,16 +49,17 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 
-class StoreApiController extends AbstractStoreController
```

Comment:
`AbstractStoreController` marked as [@internal](https://github.com/shopware/platform/blob/trunk/src/Core/Framework/Store/Api/AbstractStoreController.php#L16) and moreover we do not use its methods and shopware does not require extending any parent API controller